docs/security/key-rotation.md

```
@@ -0,0 +1,151 @@
+
+- `ACCOUNT_CIPHER_ACTIVE_KEY` is required when `ACCOUNT_CIPHER_KEYS` is set and
+  must name one of the listed versions.
+- Every key must decode to exactly 32 bytes.
+- Keys are **never** logged.
+
+## Rotation runbook
+
+Rotate to a new key **without downtime and without losing decryptability**:
+
+1. **Add the new key** alongside the current one, keeping the old one active:
+   ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)